public/c4Game.tsx

```diff
@@ -17,64 +17,94 @@ import axios from 'axios'
 stateFunctions.makeAMove(stateFunctions.state.gameId, stateFunctions.state.moves[0].x, stateFunctions.state.moves[0].y, stateFunctions.state.myColor)
 
 const config = {
-    rowHeight: 300,
-    rowWidth: 300,
-    boardWidth: 1000,
-    boardHeight: 1000,
-    gamePieceDiameter: 100
+    boardWidth: 750,
+    boardHeight: 750,
+    gamePieceDiameter: 75,
+    boarderRadius: 100
 }
 
 
-const Board = ({rows}) => {
+const Board = ({ columns }) => {
     const boardStyle = {
         width: config.boardWidth + 'px',
         height: config.boardHeight + 'px'
     }
-    const Rows = rows.map((row) => {
-        return <Row pieceArray={row}></Row>
+    const Columns = columns.map((column) => {
+        return <Column pieceArray={column}></Column>
     })
     return <div style={boardStyle} className="board">
-        {Rows}
+        {Column}
     </div>
 }
-const Row = ({ pieceArray }) => {
-    const rowStyle = {
+const Column = ({ pieceArray }) => {
+    const columnStyle = {
         width: '100%',
         display: 'align-items'
     }
-    const gamePieces = pieceArray.map((gamePiece) => {
-        return <GamePiece color={gamePiece}></GamePiece>
+
+    // 1. What type is clickChoice?  A function
+    // 2. When do you want the desired event to happen? What should trigger it?  when a user clicks a gamepiece.
+    // 3. When the event is triggered, what exactly do you want to happen? makeAMove should fire with the users choice as the input
+    debugger
+    const clickChoice = (gamePieces) => {
```

Comment:
the logic for this function is still broken.  Will need to be fixed before adding to Master.